lib/msal-node/test/client/ManagedIdentitySources/AppService.spec.ts

```diff
@@ -176,6 +178,70 @@ describe("Acquires a token successfully via an App Service Managed Identity", ()
         });
     });
 
+    describe("Miscellaneous", () => {
+        test("ignores a cached token when claims are provided and the Managed Identity does support token revocation, and ensures the token revocation query parameter token_sha256_to_refresh was included in the network request to the Managed Identity", async () => {
```

Comment:
Add test when just the capabilities are set and claims are not passed?

Reply:
Are you covering these test cases:
1. When capabilities are set but no claims are passes, the capabilities should be passed to the endpoint. No refresh of cache on 2nd request when claims are not passed.
2. When claims are passed but capabilities is not set, the cache is still refreshed and the claims are passed to the endpoint.
3. When both are present, this you already have covered.

Reply:
This test covers both 1 and 3. I made a new test for 2.

Reply:
This test covers both 1 and 3. I made a new test for 2.